core/domain/topic_services.py

```diff
@@ -1688,39 +1688,44 @@ def get_topic_id_to_topic_name_dict(topic_ids: List[str]) -> Dict[str, str]:
     return topic_id_to_topic_name
 
 
-def update_chapters_counts_in_topic_summaries(
+def get_chapters_counts_in_topic_summaries(
     topic_summary_dicts: List[topic_domain.FrontendTopicSummaryDict]
-) -> List[topic_domain.FrontendTopicSummaryDict]:
-    """Returns topic summary dicts after updating chapter counts in them.
+) -> Dict[str, topic_domain.TopicChapterCounts]:
```

Comment:
You call the returned thing "chapter counts" but the function name is "chapters counts" -- please be consistent.